cmd/server/assets/realm.html

```diff
@@ -172,6 +172,32 @@ <h1>Realm settings</h1>
               </select>
             </div>
           </div>
+
+          <div class="form-group row">
+            <label for="pwdRotate" class="col-sm-3">Password rotation every:</label>
+            <div class="col-sm-2">
+              <select class="form-control" name="pwdRotate" id="pwdRotate">
+                {{$current := $realm.PasswordRotationPeriodDays}}
+                {{range $prd := .pwdRotateDays}}
+                <option value="{{$prd}}" {{if (eq $prd $current)}}selected{{end}}>{{if (eq $prd 0)}}Off{{else}}{{$prd}}{{end}}</option>
+                {{end}}
+              </select>
+            </div>
+            <div class="col-sm-1">days</div>
+          </div>
+
+          <div class="form-group row">
+            <label for="pwdWarn" class="col-sm-3">Password rotation warning:</label>
```

Comment:
Is this days before expiration or days since creation? What's the anchor point and how is it calculated?

Reply:
anchor point is days before expiration

Reply:
It'd be good to have some help text for the user to explain this.